packages/local_auth/local_auth_android/android/src/main/java/io/flutter/plugins/localauth/LocalAuthPlugin.java

```diff
@@ -286,19 +312,28 @@ private ArrayList<String> getEnrolledBiometrics() {
     return biometrics;
   }
 
-  private boolean isDeviceSupported() {
+  @VisibleForTesting
+  public boolean isDeviceSupported() {
     if (keyguardManager == null) return false;
-    return (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M && keyguardManager.isDeviceSecure());
+    return (Build.VERSION.SDK_INT >= 23 && keyguardManager.isDeviceSecure());
   }
 
   private boolean canAuthenticateWithBiometrics() {
     if (biometricManager == null) return false;
-    return biometricManager.canAuthenticate() == BiometricManager.BIOMETRIC_SUCCESS;
+    return biometricManager.canAuthenticate(BiometricManager.Authenticators.BIOMETRIC_WEAK)
```

Comment:
Shouldn't we pass `WEAK | STRONG` in these?

Reply:
It says [here](https://developer.android.com/reference/android/hardware/biometrics/BiometricManager#canAuthenticate()) in the documentation that these are equal because it checks for at least the specified strength.